ecommerce/frontend/src/components/layout/Header.js

```diff
@@ -72,14 +72,16 @@ export const CustomerHeaderContent = () => {
 
     return (
         <div className="header-customer">
-            <Link to="/shoppingCart" className="header-customer-cart">
-                <Button
-                    className="header-customer-cart"
-                    icon={<ShoppingCartOutlined className="header-customer-cart-icon" />}
-                    ghost>
-                    My Cart
-                </Button>
-            </Link>
+            {!isVendor && (
+                <Link to="/shoppingCart" className="header-customer-cart">
+                    <Button
+                        className="header-customer-cart"
+                        icon={<ShoppingCartOutlined className="header-customer-cart-icon" />}
+                        ghost>
+                        My Cart
+                    </Button>
+                </Link>
```

Comment:
for that previous button i mentioned (the one with href), maybe you can use the same code here?